friendshipper/src/lib/components/preferences/PreferencesModal.svelte

```diff
@@ -226,6 +235,20 @@
 				For engineers. Enable if you want to debug the game client locally. Increases download size.
 			</Tooltip>
 		</div>
+		{#if $dynamicConfig.playtestRegions.length > 1}
```

Comment:
If we only had one playtest region, should we still show it in preferences so users know which region they're in?

Reply:
I think I could go either way. In my head if there's only one region, I don't think the info is useful for typical users.

Reply:
gotcha, we can keep as is then